### ISSUE TITLE
Bug in python_rehearsal when loading _solutions/python_rehearsal4.py

### ISSUE THREAD

**Issue by Beramos** [jorisvandenbossche/DS-python-data-analysis#106]
There is a line in the python_reheasal notebook that throws an error. Not sure if this is done on purpose to illustrate something or if this really is a bug 

First time running the cell gives this:
```python
# %load _solutions/python_rehearsal4.py
from barometric_formula import barometric_formula
```

The second time running the cell yields the following error trace
```python
# %load _solutions/python_rehearsal4.py
from barometric_formula import barometric_formula
```
> ---------------------------------------------------------------------------
> ModuleNotFoundError                       Traceback (most recent call last)
> <ipython-input-15-be2beee7170d> in <module>()
>       1 # %load _solutions/python_rehearsal4.py
> ----> 2 from barometric_formula import barometric_formula
> 
> ModuleNotFoundError: No module named 'barometric_formula'

**Comment by jorisvandenbossche** [comment]
If I remember correctly, this line is meant to illustrate that you can also put your function definition in a separate file outside of the notebook, and import it from there. 
But of course, that only works if you actually create this file (which is not clear from the notebook that this is goal in that step). So we should add some extra comment there that we are going to put it in a file.

**Comment by stijnvanhoey** [comment]
Fixed in #108, closing issue
